packages/wallet/backend/src/gatehub/client.ts

```diff
@@ -178,6 +179,22 @@ export class GateHubClient {
     return response
   }
 
+  /**
+   * The meta was createad as `meta.meta.[property]`
+   * We should be aware of this when the user signs up (for production)
+   */
+  async updateMetaForManagedUser(
+    userUuid: string,
+    meta: Record<string, string>
+  ): Promise<void> {
+    const url = `${this.apiUrl}/auth/v1/users/managed`
+    const body = { meta }
```

Comment:
This was the reason for the faulty `meta` object.

Reply:
Added a comment and updated the types for the `meta` object here: https://github.com/interledger/testnet/pull/1675/commits/930e7509ebc3a98425963fc6a6ef980059a27d87.